Makefile

```diff
@@ -46,6 +46,9 @@ BUILD_VM_SRC_amd64=https://cloud-images.ubuntu.com/focal/current/focal-server-cl
 BUILD_VM_SRC=$(BUILD_VM_SRC_$(ZARCH))
 
 UNAME_S := $(shell uname -s)
+HOSTARCH:=$(subst aarch64,arm64,$(subst x86_64,amd64,$(shell uname -m)))
+
+LINUXKIT_HOST_TARGET=$(shell uname -s | tr '[A-Z]' '[a-z]')/$(HOSTARCH)
```

Comment:
On a 2nd thought @vmlemon -- I really see no reason for this LINUXKIT_HOST_TARGET business. First of all. the first part is wrong -- we can only ever build for linux, so it should be:
```
LINUXKIT_HOST_TARGET=linux/$(HOSTARCH)
```

And the 2nd part is actually kind of wrong too -- since what I really need is ZARCH not HOSTARCH when I'm doing any kind of cross compile.

Reply:
I ended up pulling out ```LINUXKIT_HOST_TARGET```, and subbing it with ```LINUXKIT_PLATFORM_TARGET```, and it looks like we can still successfully do a ```gmake eve```, on the M1:

```
tyson@Tysons-MacBook-Air eve % make eve
# sample output 0.0.0-HEAD-a437e8e4-xen-amd64
Creating go builder image for user eve
[+] Building 32.9s (15/15) FINISHED                                                                                                                                  
 => [internal] load build definition from Dockerfile                                                                                                            0.1s
 => => transferring dockerfile: 1.10kB                                                                                                                          0.0s
 => [internal] load .dockerignore                                                                                                                               0.0s
 => => transferring context: 2B                                                                                                                                 0.0s
 => [internal] load metadata for docker.io/library/golang:1.16-alpine                                                                                          32.7s
 => [ 1/11] FROM docker.io/library/golang:1.16-alpine@sha256:0dc62c5cc2d97657c17ff3bc0224214e10226e245c94317e352ee8a2c54368b4                                   0.0s
 => CACHED [ 2/11] RUN apk add --no-cache openssh-client git gcc linux-headers libc-dev util-linux libpcap-dev bash vim make protobuf protobuf-dev sudo tar cu  0.0s
 => CACHED [ 3/11] RUN deluser eve ; delgroup eve || :                                                                                                          0.0s
 => CACHED [ 4/11] RUN sed -ie /:1001:/d /etc/passwd /etc/shadow ; sed -ie /:1001:/d /etc/group || :                                                            0.0s
 => CACHED [ 5/11] RUN addgroup -g 1001 eve && adduser -h /home/eve -G eve -D -H -u 1001 eve                                                                    0.0s
 => CACHED [ 6/11] RUN echo "eve ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/eve                                                                                   0.0s
 => CACHED [ 7/11] RUN go get -u github.com/golang/protobuf/protoc-gen-go                                                                                       0.0s
 => CACHED [ 8/11] RUN go get -u gotest.tools/gotestsum                                                                                                         0.0s
 => CACHED [ 9/11] RUN go get -u github.com/seamia/protodot                                                                                                     0.0s
 => CACHED [10/11] RUN mv /go/bin/protodot /usr/local/bin                                                                                                       0.0s
 => CACHED [11/11] RUN mv /go/bin/* /usr/bin                                                                                                                    0.0s
 => exporting to image                                                                                                                                          0.0s
 => => exporting layers                                                                                                                                         0.0s
 => => writing image sha256:2fe5dea77cbc76cf55c6483583485d823f0c78cf5e9df16354a80500470a623e                                                                    0.0s
 => => naming to docker.io/library/eve-build-eve                                                                                                                0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
eve-build-eve docker container is ready to use
Cloning into '/tmp/linuxkit'...
Note: switching to '6312d580324ee5592dbd026610e1e7897a4aac6b'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6312d5803 Merge pull request #3659 from edvinerikson/fix-install
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-uefi:afc04f83065546ee92bb7f3ea436575563d99821-arm64 OVMF.fd
Unable to find image 'lfedge/eve-uefi:afc04f83065546ee92bb7f3ea436575563d99821-arm64' locally
afc04f83065546ee92bb7f3ea436575563d99821-arm64: Pulling from lfedge/eve-uefi
b2a37e724af5: Pulling fs layer
b2a37e724af5: Download complete
b2a37e724af5: Pull complete
Digest: sha256:a177876af6b116ba3e61e7d34689555183b21f7ef1db99535b82dd402b6675a8
Status: Downloaded newer image for lfedge/eve-uefi:afc04f83065546ee92bb7f3ea436575563d99821-arm64
4317387f552deac66195ecff688843f048200635e46aed3b6221807d268322e3
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-uefi:afc04f83065546ee92bb7f3ea436575563d99821-arm64 OVMF_CODE.fd
tar: OVMF_CODE.fd: Not found in archive
tar: Error exit delayed from previous errors.
ff8db96bfa6c7c119b4be48b7f3b5da7eed583f1c6e5b935d7fe03e5317fe1ad
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/firmware/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-uefi:afc04f83065546ee92bb7f3ea436575563d99821-arm64 OVMF_VARS.fd
fae10f883184586ba1cdbd6e3802eed11aa234ee1cc6a22553c67a7bea913e4b
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-grub:cb36605e5552d770166d72d1c4f28ed4848bf526-arm64 EFI
Unable to find image 'lfedge/eve-grub:cb36605e5552d770166d72d1c4f28ed4848bf526-arm64' locally
cb36605e5552d770166d72d1c4f28ed4848bf526-arm64: Pulling from lfedge/eve-grub
34be7d3f14e1: Already exists
0e6706e77798: Pulling fs layer
750021581fb8: Pulling fs layer
750021581fb8: Verifying Checksum
750021581fb8: Download complete
0e6706e77798: Download complete
0e6706e77798: Pull complete
750021581fb8: Pull complete
Digest: sha256:45666e02318d8412138090032243f5b15e8a65d253e5ed80e054c7cac81a053f
Status: Downloaded newer image for lfedge/eve-grub:cb36605e5552d770166d72d1c4f28ed4848bf526-arm64
7fb5a05a4fdc45daecf69572fb4ef1f9dd8fa24b557c4c3edfe4622e501d3f27
./tools/makeconfig.sh /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/config.img "0.0.0-M1-Mac-SC3-6cecddac" conf/DevicePortConfig conf/authorized_keys conf/grub.cfg conf/onboard.cert.pem conf/onboard.key.pem conf/root-certificate.pem conf/server conf/v2tlsbaseroot-certificates.pem
Unable to find image 'lfedge/eve-mkconf:ce8fb8d317d682b7e532e7036f356867f70ca1fa' locally
ce8fb8d317d682b7e532e7036f356867f70ca1fa: Pulling from lfedge/eve-mkconf
ab2617240c09: Pull complete 
Digest: sha256:d76f04238fa4816df57b0e04bfb3a7c57446364acb98a00b7154c6f9146b7fdb
Status: Downloaded newer image for lfedge/eve-mkconf:ce8fb8d317d682b7e532e7036f356867f70ca1fa
1024+0 records in
1024+0 records out
mkfs.fat 4.1 (2017-01-24)
/config.img has 64 heads and 32 sectors per track,
hidden sectors 0x0000;
logical sector size is 512,
using 0xf8 media descriptor, with 2048 sectors;
drive number 0x80;
filesystem has 2 12-bit FATs and 4 sectors per cluster.
FAT size is 2 sectors, and provides 502 clusters.
There is 1 reserved sector.
Root directory contains 512 slots and uses 32 sectors.
Volume ID is fe690312, volume label EVE        .
# 1M of zeroes should be enough to trigger filesystem wipe on first boot
dd if=/dev/zero bs=1048576 count=1 >> /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/persist.img
1+0 records in
1+0 records out
1048576 bytes transferred in 0.000240 secs (4367474192 bytes/sec)
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64 initrd.img
Unable to find image 'lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64' locally
419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64: Pulling from lfedge/eve-mkimage-raw-efi
3297b93f26c1: Pulling fs layer
3297b93f26c1: Verifying Checksum
3297b93f26c1: Download complete
3297b93f26c1: Pull complete
Digest: sha256:fab9cecddc61e9565df10da1046aebfc15f2d580812d78b5219640a490291770
Status: Downloaded newer image for lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64
91e35a923531322dc5bd185085bfe7348cdf89a10c8a96c67518ca4f2ab3b95a
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64 installer.img
e7d44464c43d9b0eddd0405e8002004c214fbf7edaa7bd7bcd62c722534e90b3
Done building /Users/tyson/X3/eve/build-tools/bin/linuxkit
./tools/makerootfs.sh images/rootfs-kvm.yml /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/rootfs.img squash arm64
Extract kernel image: docker.io/lfedge/eve-kernel:c72ae5e9e214d0314863279b7b8662a8765afff3-arm64
Image docker.io/lfedge/eve-kernel:c72ae5e9e214d0314863279b7b8662a8765afff3-arm64 not found in local cache, pulling
Unable to find image 'lfedge/eve-mkrootfs-squash:550074297c15f9e925a16025a0fc5c5c962f3fa1' locally
550074297c15f9e925a16025a0fc5c5c962f3fa1: Pulling from lfedge/eve-mkrootfs-squash
e0d6c1806eaa: Pull complete 
Digest: sha256:45020bdee0c623d450fce9acfd9fa9d2790d852e312deb8a77752b2790d81aa7
Status: Downloaded newer image for lfedge/eve-mkrootfs-squash:550074297c15f9e925a16025a0fc5c5c962f3fa1
Add init containers:
Process init image: docker.io/linuxkit/init:a68f9fa0c1d9dbfc9c23663749a0b7ac510cbe1c
Process init image: docker.io/linuxkit/runc:f79954950022fea76b8b6f10de58cb48e4fb3878
Process init image: docker.io/linuxkit/containerd:1ae8f054e9fe792d1dbdb9a65f1b5e14491cb106
Process init image: docker.io/linuxkit/getty:v0.5
Process init image: docker.io/linuxkit/memlogd:v0.5
Process init image: docker.io/lfedge/eve-dom0-ztools:7bf5b83afe9de4e93e5435316a1af53e5a6af0bd-arm64
Image docker.io/lfedge/eve-dom0-ztools:7bf5b83afe9de4e93e5435316a1af53e5a6af0bd-arm64 not found in local cache, pulling
Process init image: docker.io/lfedge/eve-grub:cb36605e5552d770166d72d1c4f28ed4848bf526-arm64
Process init image: docker.io/lfedge/eve-fw:704d83d6714643770d9496365435cf833ec7c5e6-arm64
Process init image: docker.io/lfedge/eve-xen:7c735a8b2da8edebdc36a3885b6e1c4923bd03e7-arm64
Image docker.io/lfedge/eve-xen:7c735a8b2da8edebdc36a3885b6e1c4923bd03e7-arm64 not found in local cache, pulling
Process init image: docker.io/lfedge/eve-gpt-tools:11cd917db4f3b5d12bec731926af2832d52f5362-arm64
Image docker.io/lfedge/eve-gpt-tools:11cd917db4f3b5d12bec731926af2832d52f5362-arm64 not found in local cache, pulling
Add onboot containers:
  Create OCI config for lfedge/eve-rngd:de0b4f4fa3638ad1c62d75618523a4e6590db27e-arm64
Image docker.io/lfedge/eve-rngd:de0b4f4fa3638ad1c62d75618523a4e6590db27e-arm64 not found in local cache, pulling
  Create OCI config for linuxkit/sysctl:v0.5
  Create OCI config for linuxkit/modprobe:v0.5
  Create OCI config for lfedge/eve-storage-init:5098d4cfee893f1f71dc927a8895fd5feafd428f-arm64
Image docker.io/lfedge/eve-storage-init:5098d4cfee893f1f71dc927a8895fd5feafd428f-arm64 not found in local cache, pulling
Add service containers:
  Create OCI config for lfedge/eve-newlog:2cba218b037b8ac90edc6435da5f73876e677709-arm64
Image docker.io/lfedge/eve-newlog:2cba218b037b8ac90edc6435da5f73876e677709-arm64 not found in local cache, pulling
  Create OCI config for linuxkit/openntpd:v0.5
  Create OCI config for lfedge/eve-debug:7922697ac4fc5d8795fc2535ff5ef6d6cfd9333e-arm64
Image docker.io/lfedge/eve-debug:7922697ac4fc5d8795fc2535ff5ef6d6cfd9333e-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-wwan:ba718e0f61fcd924ce4a6438be1b263395ea8ef3-arm64
Image docker.io/lfedge/eve-wwan:ba718e0f61fcd924ce4a6438be1b263395ea8ef3-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-wlan:50c41b2b45c65411b658de758b92d25a17ce2835-arm64
Image docker.io/lfedge/eve-wlan:50c41b2b45c65411b658de758b92d25a17ce2835-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-guacd:f12cf79b6d3e07b5fafb015ae6cf134322725abb-arm64
Image docker.io/lfedge/eve-guacd:f12cf79b6d3e07b5fafb015ae6cf134322725abb-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-pillar:9b095f7e6b9d1252c8896dcc542618299782207b-arm64
Image docker.io/lfedge/eve-pillar:9b095f7e6b9d1252c8896dcc542618299782207b-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-vtpm:70ef80e77ff408081faae824e28b23bedaa7f028-arm64
Image docker.io/lfedge/eve-vtpm:70ef80e77ff408081faae824e28b23bedaa7f028-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-watchdog:30efa1a5dd64264aabe5b98c2a581d68bbfb3e2c-arm64
Image docker.io/lfedge/eve-watchdog:30efa1a5dd64264aabe5b98c2a581d68bbfb3e2c-arm64 not found in local cache, pulling
  Create OCI config for lfedge/eve-xen-tools:c8fa278e331dc617fabffcbae9fa44c6f04749e4-arm64
Image docker.io/lfedge/eve-xen-tools:c8fa278e331dc617fabffcbae9fa44c6f04749e4-arm64 not found in local cache, pulling
Add files:
  /etc/eve-release
Parallel mksquashfs: Using 4 processors
Creating 4.0 filesystem on /rootfs.img, block size 131072.
[===========================================================/] 11660/11660 100%

Exportable Squashfs 4.0 filesystem, gzip compressed, data block size 131072
	compressed data, compressed metadata, compressed fragments,
	compressed xattrs, compressed ids
	duplicates are removed
Filesystem size 203131.13 Kbytes (198.37 Mbytes)
	32.53% of uncompressed filesystem size (624439.26 Kbytes)
Inode table size 146176 bytes (142.75 Kbytes)
	25.40% of uncompressed inode table size (575598 bytes)
Directory table size 154672 bytes (151.05 Kbytes)
	49.21% of uncompressed directory table size (314338 bytes)
Number of duplicate files found 1785
Number of inodes 14901
Number of files 7705
Number of fragments 515
Number of symbolic links  5446
Number of device nodes 46
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 1704
Number of ids (unique uids + gids) 4
Number of uids 2
	unknown (0)
	unknown (1001)
Number of gids 4
	unknown (0)
	unknown (42)
	unknown (5)
	unknown (1001)
size of /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/rootfs.img is  208007168B
mkdir -p /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/
cd /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/installer/ && _() { C=`docker create $1 fake` ; shift ; docker export $C | tar -xf - "$@" ; docker rm $C ; } ; _ lfedge/eve-u-boot:ee9d16530d75a92cd3f79ec2aa7eba2f0746f32c-arm64 boot
Unable to find image 'lfedge/eve-u-boot:ee9d16530d75a92cd3f79ec2aa7eba2f0746f32c-arm64' locally
ee9d16530d75a92cd3f79ec2aa7eba2f0746f32c-arm64: Pulling from lfedge/eve-u-boot
e767da2fc7d6: Pulling fs layer
535d04ff1a84: Pulling fs layer
e767da2fc7d6: Verifying Checksum
e767da2fc7d6: Download complete
e767da2fc7d6: Pull complete
535d04ff1a84: Download complete
535d04ff1a84: Pull complete
Digest: sha256:e150cf2de52fe608606917f43e9f579ad810dd296ad6ac12f2837a4af063cacd
Status: Downloaded newer image for lfedge/eve-u-boot:ee9d16530d75a92cd3f79ec2aa7eba2f0746f32c-arm64
2014c3ac8e5af827ab0fc5a877529f9d5dfe7f52469268c6f809d86a594c8d5d
cp pkg/eve/runme.sh /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/runme.sh
cp pkg/eve/build.yml /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/build.yml
cp images/*.yml /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac
DOCKER_ARCH_TAG=arm64 ./tools/parse-pkgs.sh pkg/eve/Dockerfile.in > /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac/Dockerfile
/Users/tyson/X3/eve/build-tools/bin/linuxkit  pkg build --platforms darwin/arm64 --hash-path /Users/tyson/X3/eve --hash 0.0.0-M1-Mac-SC3-6cecddac-kvm   --force /Users/tyson/X3/eve/dist/arm64/0.0.0-M1-Mac-SC3-6cecddac
Building "lfedge/eve:0.0.0-M1-Mac-SC3-6cecddac-kvm"
building docker.io/lfedge/eve:0.0.0-M1-Mac-SC3-6cecddac-kvm
Building for arch arm64 as lfedge/eve:0.0.0-M1-Mac-SC3-6cecddac-kvm-arm64
building for platform linux/arm64 using builder linuxkit
[+] Building 102.2s (23/23) FINISHED                                                                                                                                 
 => [internal] load remote build context                                                                                                                        0.0s
 => copy /context /                                                                                                                                             0.6s
 => [internal] load metadata for docker.io/lfedge/eve-mkconf:ce8fb8d317d682b7e532e7036f356867f70ca1fa-arm64                                                     3.3s
 => [internal] load metadata for docker.io/lfedge/eve-alpine:6.7.0                                                                                              4.1s
 => [internal] load metadata for docker.io/lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64                                            3.2s
 => [internal] load metadata for docker.io/lfedge/eve-mkimage-iso-efi:908e9e20d79019fa03ff23c4ac272c06e3e4fed8-arm64                                            4.0s
 => [internal] load metadata for docker.io/lfedge/eve-ipxe:31e7bf9b557afc425b4b171152cceb9f6d17511f-arm64                                                       4.1s
 => [raw 1/2] FROM docker.io/lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64@sha256:fab9cecddc61e9565df10da1046aebfc15f2d580812d78b5  5.8s
 => => resolve docker.io/lfedge/eve-mkimage-raw-efi:419d47e1db877f42a5a6ce4ea8740b62a67556b6-arm64@sha256:fab9cecddc61e9565df10da1046aebfc15f2d580812d78b52196  0.0s
 => => sha256:3297b93f26c17f9fce49244cad5126d2f2c68440d3051e5dee03e5d0ea6775e0 20.95MB / 20.95MB                                                                5.6s
 => => extracting sha256:3297b93f26c17f9fce49244cad5126d2f2c68440d3051e5dee03e5d0ea6775e0                                                                       0.3s
 => [ipxe 1/1] FROM docker.io/lfedge/eve-ipxe:31e7bf9b557afc425b4b171152cceb9f6d17511f-arm64@sha256:bee1193a76c99cd30a0aac6c2f2aaa8b31ce13ef2d983668221200c51d  2.5s
 => => resolve docker.io/lfedge/eve-ipxe:31e7bf9b557afc425b4b171152cceb9f6d17511f-arm64@sha256:bee1193a76c99cd30a0aac6c2f2aaa8b31ce13ef2d983668221200c51ddd92f  0.0s
 => => sha256:aa4d9fc4fefeee8933850ebad34ece386d0414b65778b937561518d1f31fae84 478.02kB / 478.02kB                                                              2.4s
 => => extracting sha256:aa4d9fc4fefeee8933850ebad34ece386d0414b65778b937561518d1f31fae84                                                                       0.0s
 => [tools 1/2] FROM docker.io/lfedge/eve-alpine:6.7.0@sha256:8756702712b3f92d4812524baa8bfa29e27a795f91f157cc9b85dc7da5121f77                                 82.0s
 => => resolve docker.io/lfedge/eve-alpine:6.7.0@sha256:8756702712b3f92d4812524baa8bfa29e27a795f91f157cc9b85dc7da5121f77                                        0.0s
 => => sha256:6c2387db53991a8510a56b9bac4a2000e76f7f89cee7f993063bcdec4b4f13e4 7.07kB / 7.07kB                                                                  1.7s
 => => sha256:bbd9b87723dab362cb8f01bbb46b3c052a69e2033122e3a8c43e834b1bcce16d 314.89MB / 314.89MB                                                             79.9s
 => => sha256:d8edae61284528e39a81465d432d544fb4ab6b6073dc1df9870db4ca2c3be4b1 1.24kB / 1.24kB                                                                  1.8s
 => => sha256:46e14658c978451a1bbc305797e3b115d25e6e32a3294a15d8e822f5b56b2860 1.49kB / 1.49kB                                                                  1.8s
 => => sha256:a37b7e2b59ac60d6dfd39a0d3a19840c9f036696feda23e8e35077289725d7fc 265B / 265B                                                                      1.8s
 => => extracting sha256:a37b7e2b59ac60d6dfd39a0d3a19840c9f036696feda23e8e35077289725d7fc                                                                       0.0s
 => => extracting sha256:46e14658c978451a1bbc305797e3b115d25e6e32a3294a15d8e822f5b56b2860                                                                       0.0s
 => => extracting sha256:bbd9b87723dab362cb8f01bbb46b3c052a69e2033122e3a8c43e834b1bcce16d                                                                       2.1s
 => => extracting sha256:d8edae61284528e39a81465d432d544fb4ab6b6073dc1df9870db4ca2c3be4b1                                                                       0.0s
 => => extracting sha256:6c2387db53991a8510a56b9bac4a2000e76f7f89cee7f993063bcdec4b4f13e4                                                                       0.0s
 => [conf 1/9] FROM docker.io/lfedge/eve-mkconf:ce8fb8d317d682b7e532e7036f356867f70ca1fa-arm64@sha256:aac761e632cff3dccdb4aa5a6d7ab26a9dc7872e11c2074089496070  1.4s
 => => resolve docker.io/lfedge/eve-mkconf:ce8fb8d317d682b7e532e7036f356867f70ca1fa-arm64@sha256:aac761e632cff3dccdb4aa5a6d7ab26a9dc7872e11c2074089496070476a7  0.0s
 => => sha256:ab2617240c098cfc7a4e67e9e1f2138043a782d280ddcc2b746cc1c073dcdda8 2.78MB / 2.78MB                                                                  1.3s
 => => extracting sha256:ab2617240c098cfc7a4e67e9e1f2138043a782d280ddcc2b746cc1c073dcdda8                                                                       0.1s
 => [iso 1/1] FROM docker.io/lfedge/eve-mkimage-iso-efi:908e9e20d79019fa03ff23c4ac272c06e3e4fed8-arm64@sha256:017d63b0020b9ac715534ccb25f7bedb158dfac2c3be32bb  8.3s
 => => resolve docker.io/lfedge/eve-mkimage-iso-efi:908e9e20d79019fa03ff23c4ac272c06e3e4fed8-arm64@sha256:017d63b0020b9ac715534ccb25f7bedb158dfac2c3be32bb88d4  0.0s
 => => sha256:8df157f6bfdbe11dc05b941a4d854fc4cc8d8818a4c850bf7d054267019fc40d 1.68kB / 1.68kB                                                                  1.5s
 => => sha256:c72fd205a53f56f2c418be9ede9b5f55d9e94a7b95e6d1c3c8c624538e1f8929 8.42MB / 8.42MB                                                                  8.0s
 => => extracting sha256:c72fd205a53f56f2c418be9ede9b5f55d9e94a7b95e6d1c3c8c624538e1f8929                                                                       0.3s
 => => extracting sha256:8df157f6bfdbe11dc05b941a4d854fc4cc8d8818a4c850bf7d054267019fc40d                                                                       0.0s
 => [raw 2/2] RUN rm /initrd.img /installer.img                                                                                                                 0.2s
 => [conf 2/9] COPY --from=iso / /                                                                                                                              0.1s
 => [conf 3/9] COPY --from=raw / /                                                                                                                              0.1s
 => [conf 4/9] COPY --from=ipxe / /                                                                                                                             0.0s
 => [tools 2/2] RUN eve-alpine-deploy.sh                                                                                                                        0.3s
 => [conf 5/9] COPY --from=tools /out/ /                                                                                                                        0.1s
 => [conf 6/9] COPY installer /bits                                                                                                                             0.4s 
 => [conf 7/9] COPY runme.sh /                                                                                                                                  0.0s 
 => [conf 8/9] RUN mkdir /in /out                                                                                                                               0.1s 
 => [conf 9/9] WORKDIR /bits                                                                                                                                    0.0s 
 => exporting to oci image format                                                                                                                              14.1s 
 => => exporting layers                                                                                                                                         5.9s
 => => exporting manifest sha256:0fc308092b90c1ae313d3f2f1e399594b0cdd26aae32b7f4817109d44841c532                                                               0.0s
 => => exporting config sha256:fd66d11d8c68b7ac24463076d6415e505a933d27f64177d79c351b5da3054127                                                                 0.0s
 => => sending tarball                                                                                                                                          8.2s
Build complete, not pushing, all done.
/Users/tyson/X3/eve/build-tools/bin/linuxkit cache export --outfile - lfedge/eve:0.0.0-M1-Mac-SC3-6cecddac-kvm-arm64| docker load
2b044a458ace: Loading layer [==================================================>]  5.638MB/5.638MB
c671bf870819: Loading layer [==================================================>]  3.904MB/3.904MB
95a1d4cec963: Loading layer [==================================================>]    478kB/478kB
8d58aec925b3: Loading layer [==================================================>]  6.736MB/6.736MB
e160230bce77: Loading layer [==================================================>]  221.8MB/221.8MB
fda49bf4540e: Loading layer [==================================================>]  2.284kB/2.284kB
0488a0988e44: Loading layer [==================================================>]     116B/116B
Loaded image: lfedge/eve:0.0.0-M1-Mac-SC3-6cecddac-kvm-arm64
rm images/rootfs-kvm.yml.in
```